repository/const.go

```diff
@@ -21,7 +21,7 @@ const (
 	MetadataColumnTime   = "METADATA$TS"
 
 	queryCreateStream        = `CREATE STREAM IF NOT EXISTS %s on table %s`
-	queryCreateTrackingTable = `CREATE TABLE IF NOT EXISTS %s LIKE %s`
+	queryCreateTrackingTable = `CREATE TRANSIENT TABLE IF NOT EXISTS %s LIKE %s`
```

Comment:
Echoing @raulb's question here so we can have a discussion on this comment - why transient? 😉